README.md

```diff
@@ -3,24 +3,61 @@ barebones setup for JavaScript / Node projects replace this with a one-liner phr
 [![Codacy Badge](https://img.shields.io/badge/Code%20Quality-D-red)](https://img.shields.io/badge/Code%20Quality-D-red)
 
 
-## About
+## Description
+Felt Teachers is a platform where schools source for qualified and approved Teachers in various sectors or courses. This platform aims to provide qualified Teachers for schools to improve the educational system of the community.
 
-What is this project about. Ok to enrich here or the section above it with an image. 
+## About
+What is this project about. Ok to enrich here or the section above it with an image.
 
 Once this repo has been setup on Codacy by the TTL, replace the above badge with the actual one from the Codacy dashboard, and add the code coverage badge as well. This is mandatory
 
 This is a simple JavaScript starter repo template for setting up your project. The setup contains
 
 - Jest: For runnung tests. We strongly recommend that JavaScript projects use Jest
 
-- Eslint & Prettier: For formatting code to match ESlint AirBnB coding standard. You might need to install the ESlint and Prettier plugins for your code editor to make the most of these utilities
+- Eslint & Prettier: For formatting code to match ESlint AirBnB coding standard. You might need to install the ESlint and Prettier plugins for your code editor to make the most of these utilities.
+- Babel: Babel allow us to use the Es6 and it interpret it for  nodejs to understand. You might need to install some babel packages to make it works.
+- nodemon: nodemon is use to test the code on development, once it is on production mode nodemon is  not needed. For nodemon to work we need to install it as a devDependencies.
```

Comment:
Please see if the about can look something like this:
About
This is a solution that will help substantially increase the supply of qualified teachers in your community. Included is a plan to integrate or collaborate with international cooperations for teachers to be trained in developing countries.

This is a Laravel and React.Js project. The setup contains

Composer: For adding third party dependencies

phpunit: For runnung tests

php-cs-fixer: For formatting code to match php coding standard

About
We seek to connect stakeholders in the legal, legislative, administrative, and financial fields to the most poverty-stricken in our society by coalescing them in one platform, making it easier for knowledge to flow while preventing, tracking and reporting abuse of rights.

Note: These are two samples gotten from other teams.